packages/runtime/container-runtime/src/gc/gcTelemetryTracker.ts

```diff
@@ -0,0 +1,353 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+import { IGarbageCollectionData } from "@fluidframework/runtime-definitions";
+import { packagePathToTelemetryProperty } from "@fluidframework/runtime-utils";
+import {
+	generateStack,
+	MonitoringContext,
+	TelemetryDataTag,
+} from "@fluidframework/telemetry-utils";
+import { ICreateContainerMetadata } from "../summary";
+import {
+	disableSweepLogKey,
+	GCNodeType,
+	UnreferencedState,
+	IGarbageCollectorConfigs,
+} from "./gcDefinitions";
+import { sendGCUnexpectedUsageEvent } from "./gcHelpers";
+import { UnreferencedStateTracker } from "./gcUnreferencedStateTracker";
+
+type NodeUsageType = "Changed" | "Loaded" | "Revived";
+
+/** The event that is logged when unreferenced node is used after a certain time. */
+interface IUnreferencedEventProps
```

Comment:
Wow do these events really have like a dozen properties?  Would be good to consider moving to JSON details model. (later PR)

Reply:
I can do that as a quick follow up.